client/src/Pages/Register/index.js

```diff
@@ -0,0 +1,170 @@
+          Signin
+        </Link>
+      </Col>
+      <Col span={15} className="signup-form-container">
+        <Title className="title" level={2}>
+          Create A New Account
+        </Title>
+        {error && <Alert id="alert" message={error} type="info" showIcon />}
+        <Form
+          onFinish={onFinish}
+          initialValues={{
+            role: 'user',
+            location: 'gaza',
+          }}
+        >
+          <Form.Item
+            label="Name"
+            name="username"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your Name!',
+              },
+              {
+                pattern: /^[a-z]([-']?[a-z]+)*( [a-z]([-']?[a-z]+)*)+$/,
+                message: 'Type your fullname.',
+              },
+            ]}
+          >
+            <Input placeholder="Enter your name..." />
+          </Form.Item>
+          <Form.Item
+            label="Email"
+            name="email"
+            rules={[
+              {
+                type: 'email',
+                message: 'Please enter a valid email!',
+              },
+              {
+                required: true,
+                message: 'Please input your Email!',
+              },
+            ]}
+          >
+            <Input placeholder="Enter a valid email..." />
+          </Form.Item>
+          <Form.Item
+            label="Mobile"
+            name="mobile"
+            rules={[
+              {
+                required: true,
+                message: 'Please input mobile number!',
+              },
+            ]}
+          >
+            <Input placeholder="Enter your mobile number..." />
+          </Form.Item>
+          <Form.Item
+            label="Password"
+            name="password"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your password!',
+              },
+            ]}
+          >
+            <Input type="password" placeholder="Enter password..." />
+          </Form.Item>
+          <Form.Item
+            label="Confirm Password"
+            name="confirmPassword"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your the same password!',
+              },
+            ]}
+          >
+            <Input type="password" placeholder="Confirm the password..." />
+          </Form.Item>
+          <Form.Item
+            className="select-input"
+            name="location"
+            label="Location"
+            rules={[
+              {
+                required: true,
+              },
+            ]}
+          >
+            <Select placeholder="Location" allowClear size="small">
+              {locations.map((city, idx) => (
+                <Option value={city} key={idx}>
+                  {city.slice(0, 1).toUpperCase() + city.slice(1)}
+                </Option>
+              ))}
+            </Select>
```

Comment:
you can use the select component created and the data array too instead of this

Reply:
Yes +1